eras/babbage/impl/src/Cardano/Ledger/Babbage/Translation.hs

```diff
@@ -168,7 +169,7 @@ instance Crypto c => TranslateEra (BabbageEra c) API.PPUPState where
 
 instance Crypto c => TranslateEra (BabbageEra c) API.ProposedPPUpdates where
   translateEra _ctxt (API.ProposedPPUpdates ppup) =
-    pure $ API.ProposedPPUpdates $ fmap translatePParams ppup
+    pure $ API.ProposedPPUpdates $ fmap translatePParamsUpdate ppup
```

Comment:
So changing from Word to Byte requires changing from `PParams` to `PParamsUpdate` ? 
Just curious, but If it's too complicated to explain, don't worry, I'm missing tons of context!

Reply:
Ah maybe it's to indicate the change itself ? :thinking:

Reply:
great question!

First notice that `PParams`  and `PParamsUpdate` are type aliases:
* https://github.com/input-output-hk/cardano-ledger/blob/master/eras/babbage/impl/src/Cardano/Ledger/Babbage/PParams.hs#L137
* https://github.com/input-output-hk/cardano-ledger/blob/2acff66e84d63a81de904e1c0de70208ff1819ea/eras/babbage/impl/src/Cardano/Ledger/Babbage/PParams.hs#L83

So this function `translatePParams` was previously handling both `HKD Identity Coin` and `HKD StrictMaybe Coin`. I wasn't able to come up with a clean way of adapting the original function to handle both while trying to manipulate the `Coin`.

Reply:
Thanks  the explanation, makes sense!

Reply:
@lehins solution is cool, though, I've used it now :)